src/components/swapv2/LimitOrder/helpers.ts

```diff
@@ -138,7 +138,6 @@ export const getErrorMessage = (error: any) => {
     4001: t`User denied message signature`,
     4002: t`You don't have sufficient fund for this transaction.`,
     4004: t`Invalid signature`,
-    '-32603': t`Error occurred. Please check your device.`,
```

Comment:
what is the actual error of this code?

Reply:
![image](https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/630bc996-d2fc-4fce-9fe0-ed5b751a989d)
 many cases, I will let it show the origin error message.

Reply:
ex msg: 
<img width="465" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/858f63f4-dc8b-4c09-9713-a74a82067d9f">